examples/tvhc/tvhc.h

```diff
@@ -20,11 +20,11 @@
 #endif  // __HELPBASE_H
 
 
-const int MAXSIZE = 80;
+const int MAXSIZE = 300;
 const int MAXSTRSIZE=256;
 const int MAXHELPTOPICID=16379;
 const char commandChar[] = ".";
-const int bufferSize = 4096;
+const int bufferSize = 1000000;

```

Comment:
We would have to `ifdef` this, because 1MB will be too much for 16-bit DOS builds. But I'm very in favor of making this variable a dynamic buffer. Then we would need no `ifdef` at all. Apparently, this variable is only written to from `addToBuffer`, so it shouldn't be too hard to change this.